jetty/src/io/pedestal/http/jetty.clj

```diff
@@ -147,13 +147,16 @@
          port :port
          {:keys [ssl? ssl-port
                  h2? h2c? connection-factory-fns
-                 context-configurator configurator max-threads daemon? reuse-addr?]
+                 context-configurator context-path configurator max-threads daemon? reuse-addr?]
           :or {configurator identity
+               context-path "/"
                max-threads (max 50 (needed-pool-size))
                h2c? true
                reuse-addr? true}} :container-options} options
         thread-pool (QueuedThreadPool. ^Integer max-threads)
         server (Server. thread-pool)
+        _ (when-not (clojure.string/starts-with? context-path "/")
```

Comment:
I'm going to add a `:require` for clojure.string in another branch, even though it's technically not needed.